hw/ip/kmac/data/kmac.hjson

```diff
@@ -371,6 +371,14 @@
           '''
           tags: ["shadowed_reg_path:u_cfg_reg_shadowed.u_cfg_reg_shadowed_entropy_fast_process"]
         } // f: entropy_fast_process
+        { bits: "20"
```

Comment:
Actually, I don't know if we need the ability to disable this. IMO, it should be always on. I can imagine it has some DV implications if we make this configurable.

Reply:
In case we really need it to be configurable, I would make it at least on by default.

Reply:
It is there to disable LFSR for power saving. but it won't be big anyway... @cdgori @tjaychen What do you think?